src/providers/mission-status/README.md

```diff
@@ -0,0 +1,105 @@
+# Mission Status
+
+## Description
+
+"Mission Status" is a feature that is used to indicate the current state of a mission with regards to one or more "Mission Actions". A mission action defines a verb that may be, for example, a task for a spacecraft (such as "Drive" or "Imagery"), a change in the state of a ground system, or any other event that is relevant to the mission. Example states for a mission action might include "Go" or "No Go", indicating whether a particular action is currently cleared for execution.
+
+
+
+## Setup
+
+In order to use this feature, YAMCS MDB and `roles.yaml` must be configured properly.
```

Comment:
I'm wondering if we should update this to be in the openmct-yamcs repo as a link